src/ert/scheduler/driver.py

```diff
@@ -84,25 +84,29 @@ async def _execute_with_retry(
             stdout, stderr = await process.communicate(stdin)
 
             assert process.returncode is not None
+            outputs = (
+                f"exit code {process.returncode}, "
+                f'output: "{stdout.decode(errors="ignore").strip() or "<empty>"}", and '
+                f'error: "{stderr.decode(errors="ignore").strip() or "<empty>"}"'
+            )
             if process.returncode == 0:
-                return True, stdout.decode(errors="ignore").strip()
+                return True, outputs
```

Comment:
Apparently this changes behavior of some tests.

Reply:
Yes, I will fix those tests, but I am not sure it is a good idea to test the logging format..